homeassistant/components/apple_tv/media_player.py

```diff
@@ -173,6 +192,23 @@ def supported_features(self):
         """Flag media player features that are supported."""
         return SUPPORT_APPLE_TV
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        attrs = {}
+
+        if self.atv and self._playing:
+            all_features = self.atv.features.all_features(include_unsupported=True)
+
+            if all_features[FeatureName.Album].state == FeatureState.Available:
+                attrs[ATTR_MEDIA_ALBUM_NAME] = self._playing.album
+            if all_features[FeatureName.Artist].state == FeatureState.Available:
+                attrs[ATTR_MEDIA_ARTIST] = self._playing.artist
```

Comment:
These two attributes should be reported via the corresponding media player entity properties that exist, `media_album_name` and `media_artist`.

Reply:
Oh, I thought I was thorough when scanning for properties. I will fix that!